platform-sdk/docs/core/dynamic-address-book/roster/glossary-of-terms.md

```diff
@@ -0,0 +1,100 @@
+# Roster Glossary of Terms
+![Event Window Terminology](./EventWindowTerminology.drawio.svg)
+![Roster Terminology Diagram](./RosterTerminologyDiagram.drawio.svg)
+## Consensus Terminology
+
+### Latest Consensus Round
+
+The most recent round to have reached consensus.  (equal to the pending consensus round - 1)
+
+### Pending Consensus Round
+
+The round that the consensus algorithm is currently attempting to reach consensus on. (equal to the latest 
+consensus round + 1)
+
+### Rounds Non-Ancient
+
+A setting. The number of rounds that have reached consensus that are not ancient.  Currently set to 26 in all 
+environments. 
```

Comment:
Probably best not to put configuration values in documentation. At some point it will change and this document will be out of date.
```suggestion
A setting. The number of rounds that have reached consensus that are not ancient.
```